pack/fhv/fhvc.json

```diff
@@ -90,9 +90,407 @@
+  },
+  {
+    "back_flavor": "After more aeons of descent I saw some side passages or burrows leading from unknown recesses of blackness to this shaft of nighted mystery\n<cite>H.P. Lovecraft, \"The Festival\"</cite>",
+    "back_illustrator": "Nele Diel",
+    "back_name": "Cavern",
+    "back_traits": "Cave. Dark.",
+    "clues": 1,
+    "code": "10540",
+    "double_sided": true,
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 13,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Open Cave",
+    "pack_code": "fhvc",
+    "position": 40,
+    "quantity": 2,
+    "shroud": 3,
+    "text": "[fast] During your turn: Test [agility] (2). This test gets +1 difficulty for each ready enemy at this location. If you succeed, Open Cave loses the [[Dark]] trait until the beginning of the next investigation phase <i>(all cards at this location to use <b>(Day [day])</b> effects)</i>. (Group limit once per round.)",
+    "traits": "Cave. Dark.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The glowing blue mushroom caps palpitate as you approach the mounth of this chamber.",
+    "back_text": "As an additional cost for you to enter Fungal Cave, investigators at your location must spend 2[per_investigator] clues, as a group.",
+    "clues": 2,
+    "code": "10541",
+    "double_sided": true,
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 14,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Fungal Cave",
+    "pack_code": "fhvc",
+    "position": 41,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "[reaction] After you discover 1 or more clues from Fungal Cave: Heal 1 damage. (Group limit once per round.)\n<b>Forced</b> - At the end of the round: Each enemy at Fungal Dave heals 2 damage.",
+    "traits": "Cave. Lair. Dark.",
+    "type_code": "location",
+    "victory": 1
+  },
+  {
+    "back_link": "10542b",
+    "code": "10542a",
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 15,
+    "enemy_damage": 1,
+    "enemy_evade": 4,
+    "enemy_fight": 3,
+    "enemy_horror": 2,
+    "faction_code": "mythos",
+    "health": 5,
+    "illustrator": "Helge C. Balzer",
+    "name": "Limulus Hybrid [day]",
+    "pack_code": "fhvc",
+    "position": 42,
+    "quantity": 1,
+    "subname": "In the Light",
+    "text": "Elusive. Massive.\nLimulus Hybrid gets +3[per_investigator] health and cannot make attacks of opportunity.\n<b>Forced</b> - After Limulus Hybrid flips to this side: Deal 1[per_investigator] damage to it.",
+    "traits": "Monster. Abomination. Elite",
+    "type_code": "enemy",
+    "victory": 2
+  },
+  {
+    "code": "10542b",
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 15,
+    "enemy_damage": 2,
+    "enemy_evade": 3,
+    "enemy_fight": 4,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "health": 5,
+    "illustrator": "Helge C. Balzer",
```

Comment:
Please add a new line:

`"hidden": true,`

As for double-sided cards, we mark one side as hidden for the db. I think unhidden confuses the db, unfortunately.